src/main/webapp/app/entities/participation/participation.component.html

```diff
@@ -151,7 +163,7 @@ <h2>
                         <div class="text-right">
                             <div class="btn-group">
                                 <ng-container *ngIf="exercise?.type === ExerciseType.PROGRAMMING && exercise.isAtLeastInstructor">
-                                    <!-- TODO: In case the last result is manuel, we should warn the user that a manual result would be overridden when clicking the button -->
+                                    <!-- TODO: In case the last result is manual, we should warn the user that a manual result would be overridden when clicking the button -->
```

Comment:
I think this TODO is implemented and can be removed

Reply:
True, Thilo implemented this 2 weeks later in `programming-exercise-instructor-trigger-build-button.component.ts`. I removed the todo.